app/assets/javascripts/ama_validators/country_province_selector.js.coffee.erb

```diff
@@ -0,0 +1,33 @@
+class window.CountryProvinceSelector
+  constructor: (@country_select, @province_select) ->
+    _province_select = @province_select
+    _provinces = JSON.parse('<%= ProvinceValidator::PROV_STATE.to_json %>')
+    _countries = JSON.parse('<%= ProvinceValidator::COUNTRIES.to_json %>')
+
+    populateSelect(@country_select, _countries, false)
+
+    $(@country_select).change ->
+      province_names = _provinces[$(this).val()]
+      populateSelect(_province_select, province_names, false)
+
+window.populateSelect = (select_box, data, include_blank) ->
```

Comment:
can we make this a private method to force people to use it via an instance of `CountryProvinceSelector`?